compiler/src/main/kotlin/io/spine/protodata/AstExtension.kt

```diff
@@ -26,13 +26,15 @@
 
 @file:JvmName("Ast")
 
+@file:Suppress("TooManyFunctions")
```

Comment:
This rule shouldn't be reported for `*Extensions.kt` files (if this is Detekt's one). I suggest fix the name of the file to have plural ending.

Reply:
@yevhenii-nadtochii, thanks for these. I'll address them in the following PR.